### PR TITLE
Fix jquery version

### PR DESCRIPTION
**Pull request by hugomaiavieira**
More info: http://stackoverflow.com/q/34702284/529418

Actually, I think just updating ember it will be fixed.

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var Router = Ember.Router.extend({
 });
 
 export default Router.map(function() {
-  this.resource('matches', function() {
+  this.route('matches', { resetNamespace: true }, function() {
```
Comment (tauil) [inline app/router.js]: Why did you change this besides the `jquery` thing referenced in the description link?

Comment (hugomaiavieira) [inline app/router.js]: @tauil this should not be in this PR, should be another one. It was a mistake. It fixes a deprecation waning that the Ember Inspector alert. Whant to remove from  this PR and open a new one?

Comment (tauil) [inline app/router.js]: Do you mean that there's no more `resource` in Ember routes? I couldn't find in the docs this... can you share the link saying that it's deprecated?

Comment (hugomaiavieira) [inline app/router.js]: deprecation-warning-resources

Comment (tauil) [inline app/router.js]: :+1:

**Comment by leandrost** [comment]
Updated with ember
